robots/dissemination/cleanup.rb

```diff
@@ -26,6 +26,10 @@ def is_full_druid_tree(druid)
           truncate_druid_tree = DruidTools::AccessDruid.new(druid, workspace_root)
           (!Dir.glob(truncate_druid_tree.path).empty? && !Dir.glob(full_druid_tree.path + '*').empty?)
         end
+
+        def workflow_service
```

Comment:
shouldn't we follow the same pattern and create a `Robots::DorRepo::Dissemination::Base` that has this method?